content/posts/irods-development-update-december-2025.md

```diff
@@ -0,0 +1,270 @@
+Title: iRODS Development Update: December 2025
+Date: 2025-12-10 10:00
+Author: Kory Draughn
+Slug: irods-development-update-december-2025
+Status: published
+
+
+Hello reader,
+
+We just held our fourth and final [TRiRODS](https://irods.org/trirods) meeting of the year. Terrell Russell, Executive Director of the iRODS Consortium, gave a talk about using open-source tools to gather and present various metrics derived from the iRODS catalog. The [video](https://www.youtube.com/watch?v=IWKrd5b-57Q) is now available at YouTube.
+
+For our final development update, work on a new authentication scheme (called **irods**) has landed in upstream. It supports password hashing, session tokens, and various commands for managing user credentials. Support for the CRC64/NVME checksum algorithm has been merged too.
+
+Here's a big one. We're preparing for an initial release of the Policy Composition framework! The code has been updated to address memory issues and we're now reviewing documentation to make sure users have what they need.
+
+Testing of the Python iRODS client (PRC) is now automated via GitHub Actions. This is huge because it speeds up the release process. The current plan is to do the same for other repositories.
```

Comment:
```suggestion
Testing of the Python iRODS client (PRC) is now automated via GitHub Actions. This is huge because it will speed up the release process. The current plan is to do the same for other repositories.
```